src/components/SQForm/SQFormButton.tsx

```diff
@@ -19,7 +19,7 @@ function SQFormButton({
   title,
   type = 'submit',
   onClick,
-}: Props): JSX.Element {
+}: SQFormButtonProps): JSX.Element {
```

Comment:
Just want to call out in our consumer apps `Props` is sufficient as long as we stick to our rule of one component per file. It's not so sufficient here because we re-use the props in Storybook.

Reply:
My main concern was that this type is exposed to consumers and simply have a type named "Props" isn't sufficient.

Reply:
Nice catch. That is indeed a very valid concern!